gamemodes/terrortown/gamemode/shared/sh_shopeditor.lua

```diff
@@ -2,6 +2,11 @@
 -- @author Alf21
 -- @module ShopEditor
 
+MODE_DEFAULT = 1
+MODE_ADDED = 2
+MODE_INHERIT_ADDED = 3
+MODE_INHERIT_REMOVED = 4
```

Comment:
Just throwing in globals isn't the best. Maybe add them to `ShopEditor` (-> `shopeditor`)

Reply:
Okay, localized modes aswell.